### PR TITLE
fix: initialize max_episode_return to negative infinity instead of zero for each algorithm

### PR DESCRIPTION
**Pull request by SimonDuToit**

### CODE REVIEW

File: mava/systems/ff_ippo.py [hunk 1]
```diff
@@ -510,7 +510,7 @@ def run_experiment(_config: DictConfig) -> float:
         )
 
     # Run experiment for a total number of evaluations.
-    max_episode_return = jnp.float32(0.0)
+    max_episode_return = jnp.float32(-jnp.inf)
```
Comment (sash-a) [inline mava/systems/ff_ippo.py]: We don't actually need the `jnp.float32` part :smile: 

```suggestion
    max_episode_return = -jnp.inf
```

File: mava/systems/ff_mappo.py [hunk 2]
```diff
@@ -516,7 +516,7 @@ def run_experiment(_config: DictConfig) -> float:
         )
 
     # Run experiment for a total number of evaluations.
-    max_episode_return = jnp.float32(0.0)
+    max_episode_return = jnp.float32(-jnp.inf)
```
Comment (sash-a) [inline mava/systems/ff_mappo.py]:
```suggestion
    max_episode_return = -jnp.inf
```

File: mava/systems/rec_ippo.py [hunk 3]
```diff
@@ -679,7 +679,7 @@ def run_experiment(_config: DictConfig) -> float:
         )
 
     # Run experiment for a total number of evaluations.
-    max_episode_return = jnp.float32(0.0)
+    max_episode_return = jnp.float32(-jnp.inf)
```
Comment (sash-a) [inline mava/systems/rec_ippo.py]:
```suggestion
    max_episode_return = -jnp.inf
```

File: mava/systems/rec_mappo.py [hunk 4]
```diff
@@ -686,7 +686,7 @@ def run_experiment(_config: DictConfig) -> float:
         )
 
     # Run experiment for a total number of evaluations.
-    max_episode_return = jnp.float32(0.0)
+    max_episode_return = jnp.float32(-jnp.inf)
```
Comment (sash-a) [inline mava/systems/rec_mappo.py]:
```suggestion
    max_episode_return = -jnp.inf
```

**Review by sash-a — CHANGES_REQUESTED** [review]
Just a minor style change, thanks for finding this bug! Can you also do this for the ppo file in the advanced usage folder :smile:

**Review by RuanJohn — DISMISSED** [review]
Good catch! And great first PR :fire:

**Comment by SimonDuToit** [comment]
I've removed the float conversions and added it to the advanced usage file.

**Review by sash-a — APPROVED** [review]
:fire: :rocket: :1st_place_medal:

**Review by RuanJohn — APPROVED** [review]
Thanks @SimonDuToit :raised_hands:

**Pull request merged**